pkg/storage/scan.go

```diff
@@ -1,19 +1,57 @@
 package storage
 
 import (
-	"bytes"
 	"database/sql"
-	"encoding/json"
 	"fmt"
 	"github.com/kptm-tools/core-service/pkg/domain"
-	"time"
 )
 
-func (s *PostgreSQLStore) CreateScansTable() error {
+func (s *PostgreSQLStore) CreateScanTable() error {
 	query := `create table if not exists scans (
-      id UUID PRIMARY KEY,
-      status JSONB,
-      results JSONB,
+      id SERIAL PRIMARY KEY,
+      tenant_id UUID,
+      operator_id UUID,
+      started_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+      ended_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
```

Comment:
Scan status should go on this table, not on `scan_hosts`

```suggestion
     status VARCHAR(50) NOT NULL, -- e.g., 'pending', 'in_progress', 'completed', 'failed'
```

Reply:
ok Lucas